BetterShardsBungee/src/main/java/vg/civcraft/mc/bettershards/bungee/LobbyHandler.java

```diff
@@ -0,0 +1,77 @@
+package vg.civcraft.mc.bettershards.bungee;
+
+import java.util.concurrent.TimeUnit;
+
+import net.md_5.bungee.api.Callback;
+import net.md_5.bungee.api.ChatColor;
+import net.md_5.bungee.api.ProxyServer;
+import net.md_5.bungee.api.ServerPing;
+import net.md_5.bungee.api.chat.TextComponent;
+import net.md_5.bungee.api.config.ServerInfo;
+import net.md_5.bungee.api.connection.ProxiedPlayer;
+
+public class LobbyHandler {
+
+	public LobbyHandler() {
+		ProxyServer.getInstance().getScheduler().schedule(BetterShardsBungee.getInstance(), new Runnable() {
+			ProxyServer proxy = BetterShardsBungee.getInstance().getProxy();
+
+			@Override
+			public void run() {
+				ServerInfo lobby = proxy.getServerInfo(BetterShardsBungee.getInstance().getConfig().getString("lobby-server", ""));
+				if (lobby == null) {
+					return;
+				}
+				
+				for (ProxiedPlayer player : lobby.getPlayers()) {
+					if(QueueHandler.getServerName(player.getUniqueId()) != null){
+						continue; //Player is in the queue used for full servers, skipping him.
+					}
+					ServerInfo server = (player.getReconnectServer() != null) ? player.getReconnectServer() : proxy.getReconnectHandler().getServer(player);
+					
+					if (server == null) {
+						proxy.getLogger().warning("Couldn't find a server to send " + player.getName() + " from the lobby.");
+						return;
```

Comment:
Probably should be `continue` here; we don't want to stop checking players just because of one bad apple